hrjobcontract/templates/CRM/Hrjobcontract/Form/PayScale.tpl

```diff
@@ -34,12 +34,8 @@
         <td class="html-adjust">{$form.pay_scale.html}</td>
       </tr>
       <tr>
-        <td class="label">{$form.currency.label}</td>
-        <td class="html-adjust">{$form.currency.html}</td>
-      </tr>
-      <tr>
         <td class="label">{$form.amount.label}</td>
-        <td class="html-adjust">{$form.amount.html}</td>
+        <td class="html-adjust">{$form.currency.html} {$form.amount.html}</td>
```

Comment:
I think the image in your PR description does not reflect this change , can you please update it